rfcs/text/0017-remove-log-original.md

```diff
@@ -16,16 +16,18 @@ The removal of `log.original` will be considered a breaking change since the fie
 
 | Field Set | Field(s) |
 | --------- | -------- |
-| [`log`](0000/log.yml) | `log.original` |
-| [`event`](0000/event.yml) | `event.original` |
+| [`log`](0017/log.yml) | `log.original` |
+| [`event`](0017/event.yml) | `event.original` |
 
-<!--
-Stage 1: Describe at a high level how this change affects fields. Include new or updated yml field definitions for all of the essential fields in this draft. While not exhaustive, the fields documented here should be comprehensive enough to deeply evaluate the technical considerations of this change. The goal here is to validate the technical details for all essential fields and to provide a basis for adding experimental field definitions to the schema. Use GitHub code blocks with yml syntax formatting.
--->
+- The internal description of the field `log.original` in [`log`](0017/log.yml) should be amended by addition of a notice of deprecation and subsequently removal if/when Deprecation progresses to Removal
 
-<!--
-Memo: see https://github.com/elastic/ecs/pull/1298#discussion_r591870463
--->
+- The internal description of the field `event.original` in [`event`](0017/event.yml) should be updated to reflect the revised scope 
+
+- The [`Beats default fields inclusion list`](../../scripts/generators/beats_default_fields_whitelist.yml) list should be updated by removing `log.original` if/when Deprecation progresses to Removal
+
+- The extended description of `log.original` in the [`Log Fields documentation`](../../docs/field-details.asciidoc#field-log-original) should be amended by addition of a notice of deprecation and subsequently removal if/when Deprecation progresses to Removal
+
+- The extended description of `event.original` in the [`Event Fields documentation`](../../docs/field-details.asciidoc#field-event-original) should be amended by addition of a notice of deprecation and subsequently removal if/when Deprecation progresses to Removal
```

Comment:
Would we have anything that we'd need to deprecate for `event.original` if we remove `log.original`?

Reply:
Thanks to @jamiehynds for spotting https://github.com/elastic/ecs/pull/1298#discussion_r604750476

Reply:
`Google Cloud integration includes a var.keep_original_message (disabled by default) which populates log.original. Will need to adjust the module to populate event.original instead.`

Reply:
Ok, good catch @jamiehynds @djptek.

I don't see a need to edit the ECS description for `event.original` if we move forward with deprecating and then removing `log.original` at the next major. When we deprecate `log.original`, we can add a note `log.original's` description suggesting to migrate to `event.original`.

Thoughts?

Reply:
Agree, that was wrong, I've changed to:

- The extended description of `event.original` in the [`Event Fields documentation`](../../docs/field-details.asciidoc#field-event-original) should be amended to clarify the absorption of `log.original`